frontend/src/components/pagination.tsx

```diff
@@ -0,0 +1,101 @@
+import { MoreHorizontal } from 'lucide-react'
+import React from 'react'
+
+interface PaginationProps {
+  currentPage: number
+  totalPages: number
+  isLoaded: boolean
+  // eslint-disable-next-line no-unused-vars
+  onPageChange: (page: number) => void
```

Comment:
Do you know if there is a way to avoid this suppression -- handle it in other/better way?

Reply:
i will try it .

Reply:
we can't resolve the ESLint issue because it is triggered by another function, and due to the state change, it causes that function to re-render.